docs/learn/price-protection.md

```diff
@@ -0,0 +1,496 @@
+---
+title: "Understand Price Protection"
+---
+
+# Understand Price Protection
+
+## Price Protection Modes
+
+Bittensor clients (BTCLI and the SDK) provides three modes to give users control over how their transactions handle adverse price movements: Safe, Partial, and Unsafe.
+
+Other users' transactions can affect the token price, even while your transaction is pending. Subnet token prices may change rapidly, with significant consequences affecting your execution price and increasing slippage. These effects can be exploited by "sandwich attacks", or can result in loss of liquidity due to organic price volatility.
+
+It is therefore important to carefully manage price protection when staking and unstaking real value liquidity, i.e. on mainnet ("finney").
+
+### Strict Safe Mode (Default)
+
+In this mode, the transaction is **rejected entirely** if executing it would push the final market price beyond the specified tolerance from the price when you submitted the transaction.
```

Comment:
```suggestion
In this mode, the transaction is **rejected entirely** if executing it would push the final market price beyond the tolerance threshold from the price when you submitted the transaction. Tolerance threshold can be specified but is 5% by default.
```